### PR TITLE
fix: update sync connection frequency tooling and documentation

### PR DESCRIPTION
**Pull request by bodinsamuel**
Follow up of #1548

## Describe your changes

- Update node-client
- Update open api
- Update documentation

### CODE REVIEW

**Review by khaliqgant — CHANGES_REQUESTED** [review]
Looks good. [Node client documentation](https://docs.nango.dev/sdks/node) should be updated as well at https://github.com/NangoHQ/nango/blob/master/docs-v2/sdks/node.mdx

**Comment by bodinsamuel** [comment]
Oh good call, I didn't noticed there was a doc for the client thanks 👍🏻

**Comment by bodinsamuel** [comment]
Is this good for you @khaliqgant ? ☺️

**Review by khaliqgant — CHANGES_REQUESTED** [review]
Can you remove the whitespace changes in `[docs-v2/sdks/node.mdx](https://github.com/NangoHQ/nango/pull/1549/files#diff-202fbe3f1a0f65d56a97c3bf855140eb2c2f504e6e519589aadcfb9cff656d3e)`? That file is sensitive in how the layout should be so the whitespace there is intentional. The only diff there should be the content you added at the bottom

**Comment by bodinsamuel** [comment]
Understood, but I think those are only whitespaces at the end of the lines. 
They don't change the layout as far as I noticed, it's git (with `git config core.whitespace fix`) and the default vscode linter that removes them.

Screenshot 2024-01-24 at 15 55 36


edit: After checking locally
The only place where it was indeed not good after trimming, was `**Return value**` but it should be a heading anyway. So I added a line-break

File: docs-v2/sdks/node.mdx [hunk 1]
```diff
@@ -416,3 +417,22 @@ console.log(vars);
 ]
 */
 ```
+
+### Update sync's connection frequency
```
Comment (bastienbeurier) [inline docs-v2/sdks/node.mdx]: I would rephrase to: 
"Override a sync's frequency for a specific connection"

File: docs-v2/sdks/node.mdx [hunk 2]
```diff
@@ -416,3 +417,22 @@ console.log(vars);
...
+
+To update a sync's connection frequency, or revert to the default frequency:
```
Comment (bastienbeurier) [inline docs-v2/sdks/node.mdx]: Same here: "To override a sync's frequency for a specific connection, or revert to the default frequency, us:"

**Comment by TBonnin** [comment]
@khaliqgant I removed the whitespace changes in node.mdx

**Review by khaliqgant — APPROVED** [review]
